corehq/apps/hqwebapp/static/hqwebapp/js/ui_elements/bootstrap3/ui-element-key-val-mapping.js

```diff
@@ -39,8 +39,8 @@ hqDefine('hqwebapp/js/ui_elements/bootstrap3/ui-element-key-val-mapping', functi
         }
         return name.replace(/[^a-z0-9]/g, function (s) {
             var c = s.charCodeAt(0);
-            if (c === 32) return '-';
-            if (c >= 65 && c <= 90) return '_' + s.toLowerCase();
+            if (c === 32) { return '-'; }
```

Comment:
Not sure if we have a style guide for that but I would add the line breaks too when adding `{}`

Reply:
The closest to a _specific_ JS style guide I found (outside our linter settings, which think this is fine) is that Vellum refers to the javascript style guide from Airbnb. For what it's worth, that guide infers (but doesn't explicitly say) that single line conditionals and functions are fine. https://github.com/airbnb/javascript?tab=readme-ov-file#whitespace--block-spacing